src/universal/modules/userDashboard/components/Organizations/Organizations.js

```diff
@@ -30,9 +34,9 @@ const Organizations = (props) => {
     <UserSettingsWrapper>
       <Helmet title="My Organizations | Parabol" />
       <div className={css(styles.wrapper)}>
-        {organizations.length ?
+        {ownedOrganizations.length ?
           <Panel label="Organizations" controls={addNewOrg()}>
-            {organizations.map((organization) =>
+            {ownedOrganizations.map((organization) =>
```

Comment:
Thanks to queries instead of changefeeds, we can now differentiate between loading states & empty states. big win!

Reply:
OMG THIS IS HUGE